consensus/polybft/bridge.go

```diff
@@ -0,0 +1,116 @@
+package polybft
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/go-hclog"
+	bolt "go.etcd.io/bbolt"
+)
+
+var _ Bridge = (*bridge)(nil)
+
+// bridge is a struct that manages different bridges
+type bridge struct {
+	bridgeManagers map[uint64]BridgeManager
+}
+
+// Bridge is an interface that defines function that a bridge must implement
+type Bridge interface {
+	Close()
+	PostBlock(req *PostBlockRequest) error
+	PostEpoch(req *PostEpochRequest) error
+	BridgeBatch(pendingBlockNumber uint64) ([]*BridgeBatchSigned, error)
+	InsertEpoch(epoch uint64, tx *bolt.Tx) error
+}
+
+var _ Bridge = (*dummyBridge)(nil)
+
+type dummyBridge struct{}
+
+func (d *dummyBridge) Close()                                {}
+func (d *dummyBridge) PostBlock(req *PostBlockRequest) error { return nil }
+func (d *dummyBridge) PostEpoch(req *PostEpochRequest) error { return nil }
+func (d *dummyBridge) BridgeBatch(pendingBlockNumber uint64) ([]*BridgeBatchSigned, error) {
+	return nil, nil
+}
+func (d *dummyBridge) InsertEpoch(epoch uint64, tx *bolt.Tx) error { return nil }
+
+// newBridge creates a new instance of bridge
+func newBridge(bridgeBackend BridgeBackend,
```

Comment:
Do we need the `BridgeBackend` interface, since we are only wrapping the `Runtime` and also in the `BridgeEventManager` it is used as the `Runtime` (https://github.com/Ethernal-Tech/blade/blob/600c8e978116dde115f150642b8f58d58cad5e28/consensus/polybft/bridge_event_manager.go#L102)?